fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -1068,6 +1079,7 @@ ACTOR Future<Void> commitBatch(
 
 	self->commitBatchesMemBytesCount -= currentBatchMemBytesCount;
 	ASSERT_ABORT(self->commitBatchesMemBytesCount >= 0);
+	wait(releaseFuture);
```

Comment:
So the `wait` here is only to limit the throughput of the proxy (200MB/second)? Cause it seems like the proxy is already guaranteed to only fetch a commit version when all previous resolving batches finished.

Reply:
Oh never mind I figured it out.